src/zentables/__init__.py

```diff
@@ -13,6 +13,6 @@
     df.zen.pretty()
 """
 
-from .zentables import ZenTablesAccessor, set_options
+from .zentables import ZenTablesAccessor, set_options, _do_suppression
```

Comment:
I thought we decided _not_ to make this part of the public API?

Reply:
I thought this was already being achieved by removing `_do_suppression` from `__all__`.
If I remove `_do_suppression` from this original import, then there is no way for me to apply this function to `test_suppression.py`, because it never gets imported. 

I may have just completely misunderstood you, but I thought this was the move that was 'condoned' given the amount of testing we had to do.

Reply:
Ah, this was my bad. To test `_do_suppression` you need to do `from .zentables import _do_suppression` at the top of test scripts instead of `import zentables` which was at the beginning of my test script. Then you don't need to make `_do_suppression` accessible from the top level of the package.